libr/core/cmd_open.c

```diff
@@ -6,6 +6,7 @@
 #include "r_util.h"
 #include "r_bin.h"
 #include "r_debug.h"
+#include "r_io.h"
```

Comment:
please reduce the includes and use `<>` instead of "" (note that rbin uses io and debug uses util, so imho just including rbin and debug shuold be enough

Reply:
use <> is good, but why reduce the includes. What if r_bin changes in the future so that it no longer needs r_io?

Reply:
anything can change in the future, but for now i think its better to keep the minimum includes as possible, compilers will also appreciate doing less work